src/_output/vue/views/al-groups-view-full.vue

```diff
@@ -70,15 +73,17 @@ export default {
       shownTabs: []
     }
   },
+  watch: {
+    languageTargetIds () {
+      this.initShownTabs()

```

Comment:
I'm not sure if this is where it's happening or not, but I found some unexpected behavior related to dragging the languages and the tabs being displayed. When I drag one of the languages, it resets the tabs being displayed, even if the order is unchanged:


![out](https://user-images.githubusercontent.com/3039310/111357923-232bfd80-8660-11eb-8726-4716cf6c470a.gif)

Reply:
great - you are right - I need to add this check - thank you